.zenodo.json

```
@@ -0,0 +1,14 @@
+{
+    "title": "smesh",
+    "description": "A simple package for generating and handling unstructured triangular and polygonal meshes, written in Fortran with Julia wrappers.",
```

```suggestion
    "description": "A simple Fortran package for generating and handling unstructured triangular and polygonal meshes.",
```
I'd suggest to leave Julia out of this, since this is just the Fortran part. For the Julia part, we should probably create a separate repo, Smesh.jl, which a separate Zenodo file.